game/state/gamestate_serialize.cpp

```diff
@@ -814,6 +814,16 @@ template <> void serializeIn(const GameState *state, sp<SerializationNode> node,
 	serializeIn(state, node->getNode("num_labs_created"), r.num_labs_created);
 }
 
+template <> void serializeIn(const GameState *state, sp<SerializationNode> node, GameTime &t)
+{
+	if (!node)
+		return;
+
+	unsigned int ticks;
+	serializeIn(state, node->getNode("ticks"), ticks);
+	t = GameTime(ticks);
```

Comment:
If the node doesn't actually have a "ticks" this will have an unset 'ticks' value - I guess you either want to set 'ticks' to a sane default, or use node->getNodeReq("ticks") if it should error if the node is missing